infrastructure/alerts.tf

```diff
@@ -11,6 +11,7 @@ module "fpl-action-group" {
   short_name             = "${var.product}-support"
   email_receiver_name    = "FPL Support Mailing List"
   email_receiver_address = data.azurerm_key_vault_secret.fpl_support_email_secret.value
+  common_tags            = var.common_tags
```

Comment:
Oh actually - maybe this is the issue in the PR build atm, this module's source is different from the below ones so maybe doesn't need common_tags?

Reply:
Thanks @DanCatchpole , it fixed the pr build